libs/blocks/quiz/utils.js

```diff
@@ -35,7 +36,7 @@ export const initConfigPathGlob = (rootElement) => {
   metaData = getNormalizedMetadata(rootElement);
   shortQuiz = metaData.shortquiz?.text === 'true';
   configPath = initConfigPath(metaData);
-  quizKey = initQuizKey(rootElement);
+  quizKey = initQuizKey();
```

Comment:
This is a complete nit-pick, but given that the init methods for quizKey, analyticsType and analyticsQuiz are so trivial, why not just do away with the methods and set them directly, like we do the shortQuiz value?

export const initConfigPathGlob = (rootElement) => {
  metaData = getNormalizedMetadata(rootElement);
  shortQuiz = metaData.shortquiz?.text === 'true';
  configPath = initConfigPath(metaData);
  quizKey = metaData.storage?.text.toLowerCase();
  analyticsType = metaData.analyticstype?.text;
  analyticsQuiz = metaData.analyticsquiz?.text;
  return { configPath, quizKey, analyticsType, analyticsQuiz, shortQuiz };
};